react/src/pages/ResourcesPage.tsx

```diff
@@ -39,14 +41,11 @@ const ResourcesPage: React.FC<ResourcesPageProps> = (props) => {
       ]}
       styles={{
         body: {
-          padding: 0,
-          paddingTop: 1,
-          overflow: 'hidden',
+          padding: `${token.paddingSM}px ${token.paddingLG}px ${token.paddingLG}px ${token.paddingLG}px`,
```

Comment:
It would be good to create a PR to modify BAICard so that the bodyStyle automatically changes when a tabList is specified.